### ISSUE TITLE
Alfred 3 support

### ISSUE THREAD

**Issue by apiology** [mannieschumpert/quicktask#3]
There's a hard-coded path inside this plugin that results in a fresh install for a new user running Alfred 3 not working.  The symptom is a PHP error message when trying to run the `akey` command.

My debug log in Alfred showed:

```
[2017-06-15 20:56:49][ERROR: action.script] mkdir: /Users/broz/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data: No such file or directory
```

I worked around it with:

```sh
ln -s ~/"Library/Application Support/Alfred 3" ~/"Library/Application Support/Alfred 2"
```

**Comment by therealscottcarlow** [comment]
Yeah, this was addressed in #2 and #4 fixes it once it gets merged.

**Comment by ialexryan** [comment]
I think now that #4 is merged this issue can be closed :)

**Comment by apiology** [comment]
Hey @mannieschumpert , any chance a new release could be spun, put into Dropbox and linked to on http://mannieschumpert.com/quicktask/ ?  Looks like all the code is in place.
